api/Window.json

```diff
@@ -6093,47 +6143,97 @@
           "description": "<code>pageshow</code> event",
           "support": {
             "chrome": {
-              "version_added": null
+              "version_added": true
             },
             "chrome_android": {
-              "version_added": null
+              "version_added": true
             },
             "edge": {
-              "version_added": null
+              "version_added": "12"
             },
             "firefox": {
-              "version_added": "1.5"
+              "version_added": true
             },
             "firefox_android": {
-              "version_added": "4"
+              "version_added": true
             },
             "ie": {
-              "version_added": null
+              "version_added": true
             },
             "opera": {
-              "version_added": null
+              "version_added": true
             },
             "opera_android": {
-              "version_added": null
+              "version_added": true
             },
             "safari": {
-              "version_added": null
+              "version_added": true
             },
             "safari_ios": {
-              "version_added": null
+              "version_added": true
             },
             "samsunginternet_android": {
-              "version_added": null
+              "version_added": true
             },
             "webview_android": {
-              "version_added": null
+              "version_added": true
             }
           },
           "status": {
             "experimental": false,
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "persisted": {
```

Comment:
Like before, I don't think we typically duplicate event object properties in `Window.json` like this. I think we can remove this feature.

Reply:
Good call @ddbeck - I've removed the `persisted` event object property data from here.